### PR TITLE
M sa2 08 19

### PR DESCRIPTION
**Pull request by begun74**

### CODE REVIEW

File: Biahun_Uladzimir/02.Git.local/02.Git.local [hunk 1]
```diff
@@ -0,0 +1,67 @@
+      git init
```
Comment (pluhin) [inline Biahun_Uladzimir/02.Git.local/02.Git.local]: Where is markup here? Could you please use at least code section for your commands (add bash syntax highlighting)

**Comment by begun74** [comment]
```console
Новая версия задания
```

File: Biahun_Uladzimir/02.Git.local/02.Git.local (hunk 1 above)
Comment (begun74) [reply Biahun_Uladzimir/02.Git.local/02.Git.local]:
```
Новая версия проекта
```

File: Biahun_Uladzimir/02.Git.local/02.Git.local.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+ 1003  git init
```
Comment (pluhin) [inline Biahun_Uladzimir/02.Git.local/02.Git.local.md]: Could you please use markup here

File: Biahun_Uladzimir/02.Git.local/02.Git.local.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+ 1067* git branch -d dev
+ 1068  git branch -v
+ 1069  git checkout master
+ 1070  git merge master
+ 1071  git checkout master
+ 1072  ls -la
+ 1073  git merge dev
+ 1074  ls -la
+ 1075  history
+
```
Comment (pluhin) [inline Biahun_Uladzimir/02.Git.local/02.Git.local.md]: Could you provide list of commits in each branches

Comment (begun74) [reply Biahun_Uladzimir/02.Git.local/02.Git.local.md]: But, they are in Biahun_Uladzimir/02.Git.local/02.Git.local.md  file

Comment (begun74) [reply Biahun_Uladzimir/02.Git.local/02.Git.local.md]: git log --oneline --decorate --graph --all  ?

**Comment by begun74** [comment]
````
Add git log --oneline --decorate --graph --all 
````

File: Biahun_Uladzimir/02.Git.local/02.Git.local.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+ 1067* git branch -d dev
+ 1068  git branch -v
+ 1069  git checkout master
+ 1070  git merge master
+ 1071  git checkout master
+ 1072  ls -la
+ 1073  git merge dev
+ 1074  ls -la
+ 1075  history
+1111  git log --oneline --decorate --graph --all
```
Comment (pluhin) [inline Biahun_Uladzimir/02.Git.local/02.Git.local.md]: I need output on this command, even short `git log --oneline`
Your branches should have:
master (init) 6 commits
dev (from master)  5 commits
features/do_one (from dev) 6 commits
hotfix/we_gonna_die (from master) 3 commits

File: Biahun_Uladzimir/02.Git.local/02.Git.local.md (hunk 2 above)
Comment (pluhin) [reply Biahun_Uladzimir/02.Git.local/02.Git.local.md]: And where is markup?

File: Biahun_Uladzimir/02.Git.local/02.Git.local.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
+ 1003  git init
+ 1004  git config --global user.name "Biahun Uladzimir"
+ 1005  git config --global user.email "begun74@gmail.com"
+ 1006  git config user.name "Biahun Uladzimir"
```
Comment (pluhin) [inline Biahun_Uladzimir/02.Git.local/02.Git.local.md]: it is duplicate with 1004

File: Biahun_Uladzimir/02.Git.local/02.Git.local.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+ 1007  date >> master.txt
+ 1008  git add master.txt
+ 1009  git commit -a -m "Init master"
+ 1010   date >> master.txt
+ 1011  git commit -a -m " === 2 commit ==="
+ 1012  git log --oneline --decorate --graph --all
```
Comment (pluhin) [inline Biahun_Uladzimir/02.Git.local/02.Git.local.md]: need output

Comment (begun74) [reply Biahun_Uladzimir/02.Git.local/02.Git.local.md]: *   c84771d (HEAD, master) Merge branch 'dev'
|\
| * 5c4d5b7 (dev)  === 10 commit ===
| * e9b1746  === 9 commit ===
| * b5ff317  === 6 commit ===
| * 70f4cac  === 5 commit ===
* |   6631988 Merge branch 'hotfix/we_gonna_die'
|\ \
| * | 7b96b3c  === 13 commit ===
* | | 214eb0c  === 12 commit ===
* | | 5746a77  === 11 commit ===
|/ /
* | 15f23c8  === 8 commit ===
* | 080c739  === 7 commit ===
* | 8db6937  === 4 commit ===
* | fb87e98  === 3 commit ===
|/
* 199e494  === 2 commit ===
* 9477ce6 Init master

Comment (begun74) [reply Biahun_Uladzimir/02.Git.local/02.Git.local.md]: git log --oneline
c84771d Merge branch 'dev'
6631988 Merge branch 'hotfix/we_gonna_die'
7b96b3c  === 13 commit ===
214eb0c  === 12 commit ===
5746a77  === 11 commit ===
5c4d5b7  === 10 commit ===
e9b1746  === 9 commit ===
15f23c8  === 8 commit ===
080c739  === 7 commit ===
b5ff317  === 6 commit ===
70f4cac  === 5 commit ===
8db6937  === 4 commit ===
fb87e98  === 3 commit ===
199e494  === 2 commit ===
9477ce6 Init master

**Comment by begun74** [comment]
Excuse me May I see example markup? I don't understand

File: Biahun_Uladzimir/02.Git.local/02.Git.local.md (hunk 6 above)
Comment (begun74) [reply Biahun_Uladzimir/02.Git.local/02.Git.local.md]: Excuse me May I see example markup? I don't understand

Comment (pluhin) [reply Biahun_Uladzimir/02.Git.local/02.Git.local.md]: > Excuse me May I see example markup? I don't understand
https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet

Comment (begun74) [reply Biahun_Uladzimir/02.Git.local/02.Git.local.md]: Ok. Where his use  for example?

File: Biahun_Uladzimir/02.Git.local/README.md [hunk 7]
```diff
@@ -0,0 +1,3 @@
+````
+	Learning git. Create branches, commits and etc
```
Comment (pluhin) [inline Biahun_Uladzimir/02.Git.local/README.md]: It isn't file with git markup

File: Biahun_Uladzimir/03.Git.hosting/03.Git.hosting.md [hunk 8]
```diff
@@ -0,0 +1,5 @@
+https://gitlab.com/Begun74/test_ta
```
Comment (pluhin) [inline Biahun_Uladzimir/03.Git.hosting/03.Git.hosting.md]: markup missing here

**Pull request merged**